app/src/main/java/org/fossasia/openevent/general/event/EventDetailsViewModel.kt

```diff
@@ -255,6 +261,50 @@ class EventDetailsViewModel(
             })
     }
 
+    fun syncTickets(event: Event) {
+        compositeDisposable += ticketService.syncTickets(event.id)
+            .withDefaultSchedulers()
+            .subscribe({
+                loadPriceRange(it, event.paymentCurrency.toString())
+            }, {
+                Timber.e(it, "Error fetching tickets")
+            })
+    }
+
+    private fun loadPriceRange(tickets: List<Ticket>, paymentCurrency: String) {
+        var minPrice = tickets[0].price
```

Comment:
What happens if an event doesn't have ticket yet?

Reply:
Will return if the list is empty :+1:

Reply:
Updated